src/Livewire/Forms/Contact.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace FluxErp\Livewire\Forms;
+
+use FluxErp\Actions\Address\CreateAddress;
+use FluxErp\Actions\Address\UpdateAddress;
+use FluxErp\Actions\Contact\CreateContact;
+use FluxErp\Actions\Contact\UpdateContact;
+use FluxErp\Models\Client;
+use FluxErp\Models\Language;
+use Illuminate\Support\Arr;
+use Livewire\Form;
+
+class Contact extends Form
+{
+    public ?int $id = null;
+
+    public ?int $client_id = null;
+
+    public ?string $company = null;
+
+    public ?string $salutation = null;
+
+    public ?string $title = null;
+
+    public ?string $firstname = null;
+
+    public ?string $lastname = null;
+
+    public ?string $street = null;
+
+    public ?int $countryId = null;
+
+    public ?string $zip = null;
+
+    public ?string $city = null;
+
+    public ?int $language_id = null;
+
+    public function save(): void
+    {
+        $contact = Arr::only($this->toArray(), ['id', 'client_id']);
+        $action = $this->id ? UpdateContact::make($contact) : CreateContact::make($contact);
+        $response = $action->checkPermission()
```

Comment:
this can throw an Unauthorized exception, which is not caught anywhere, or am I wrong?

Reply:
is catched in ContactList.php